app/src/main/java/com/android/voyageur/ui/notifications/AndroidStringProvider.kt

```diff
@@ -0,0 +1,7 @@
+package com.android.voyageur.ui.notifications
+
+import android.content.Context
+
+class AndroidStringProvider(private val context: Context) : StringProvider {
```

Comment:
I would suggest to add documentation here as well

Reply:
Done